packages/backend-modules/access/graphql/schema-types.js

```diff
@@ -34,6 +34,12 @@ type AccessCampaign {
   endAt: DateTime!
 }
 
+type AccessInfo {
```

Comment:
Can we use a less general name for the query? `accessinfo` could mean any info related to access of any content :D. 
As the content references granter, I'd go with `granterInfo` or `accessGraterInfo`